docs/requirements.txt

```diff
@@ -4,3 +4,4 @@ sphinx-tabs==3.2.0
 sphinx-rtd-theme==0.5.2
 sphinxemoji==0.1.8
 sphinx-autoapi==1.8.4
+sphinx-notfound-page
```

Comment:
I think we want to test the current version of the extension with the docs, not the one from pypi. This can be done by injecting the package in https://github.com/readthedocs/sphinx-notfound-page/blob/0217cb1e628140744559213a105642a6c9f7ed49/docs/conf.py#L17 or by adding `-e ..` here.

Reply:
I thought the same. However, `pip install .` is executed first in our RTD commands. So, when this is called it says the requirement is already satisfied and does nothing.